### ISSUE TITLE
Links to a dissertation: personal homepage or university library?

### ISSUE THREAD

**Issue by julbinb** [nuprl/website#213]
Jonathan @schuster suggested that for the links to dissertations on the people page, it might be better to use the university digital library rather than personal homepages. I agree with that. 

What do other folks think?

**Comment by ulysses4ever** [comment]
I don't believe too much in any kind of homebrew systems so I'd go with whatever, prioritizing links to GitHub and DOI's.

**Comment by bennn** [comment]
+1 for university links

**Comment by LeifAndersen** [comment]
A lot of people make both links available, doing something like:

```
<Paper Name> [Author's Version]
```

Where `<Paper Name>` links to the official version, and `Author's Version` links to whatever the author has on their website. This also has the advantage of allowing the author to have an expanded version available.

**Comment by ulysses4ever** [comment]
SGTM

**Comment by julbinb** [comment]
Oh, that's a great idea, thanks! CC current people page manager @donald-pinckney
